src/sql/platform/accounts/common/accountStore.ts

```diff
@@ -47,6 +47,15 @@ export default class AccountStore implements IAccountStore {
 		});
 	}
 
+	public removeAllAccounts(): Thenable<boolean> {
+		return this.doOperation(() => {
```

Comment:
Any reason not to make this an async function?

Reply:
Just following the paradigm of this file as well. Do you think I should keep doing that or just use async/await?

Reply:
I'm fine either way but in general I would rather see new stuff written using async/await so people get more in the habit of it.

Reply:
Done :)